src/main/webapp/script.js

```diff
@@ -62,50 +71,22 @@ function fetchBlobstoreUrl() {
 
 // Enables or disables the editable form in the profile page.
 function toggleProfileEditMode() {
-  const staticEl = document.getElementById('static');
-  const editableEl = document.getElementById('editable');
+  const staticEl = document.getElementById('static-user-info');
+  const editableEl = document.getElementById('user-form');
   const buttonEl = document.getElementById('edit-button');
 
-  if(staticEl.classList.contains('hidden')) {
-    staticEl.classList.remove('hidden');
-    editableEl.classList.add('hidden');
+  if(staticEl.classList.contains('d-none')) {
+    staticEl.classList.remove('d-none');
+    editableEl.classList.add('d-none');
     buttonEl.innerHTML = 'Edit Profile';
   }
   else {
-    staticEl.classList.add('hidden');
-    editableEl.classList.remove('hidden');
+    staticEl.classList.add('d-none');
+    editableEl.classList.remove('d-none');
     buttonEl.innerHTML = 'Back';
```

Comment:
Should this be 'back' or something like 'view'. 'Back' means that no change was made/ changes were discarded.

Reply:
Actually, that's exactly what the button does. If you click back, you are discarding your changes. If you click submit, the changes will be saved.